templates/vue/src/components/Lightbox/index.vue

```diff
@@ -82,10 +87,11 @@ export default {
       },
       showCompletionScreen: false,
       rowRefs: [],
+      selectedTab: "",
     }
   },
   computed: {
-    ...mapState(["h5pSettings", "rootId"]),
+    ...mapState(["h5pSettings", "rootId", "displayTydeMode"]),
```

Comment:
Rather than have this in the store, can we just have a computed property in this component? We do not seem to be using this anywhere else for the most part?